### PR TITLE
Raw and binary strings

### PR DESCRIPTION
**Pull request by listochkin**

### CODE REVIEW

File: presentations/strings/7.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fn main () {
+    let json = r##"
+{
```
Comment (jonathanpallant) [inline presentations/strings/7.rs:3]: Maybe this block should be indented to line it up with the rest of the code. It looks tidier, but the indentation will be part of the string.

Comment (listochkin) [reply presentations/strings/7.rs:3]: I thought about it, too, but decided against it to show that multiline strings in Rust behave like heredocs in many other languages. A presenter should mention it, and I thought that this unpleasant formatting will prompt them to do so. I've added some text to a slide to mention it.

File: presentations/strings/slides.adoc [hunk 2]
```diff
@@ -111,3 +111,25 @@ It's possible to accept either rather painlessly:
...
+
+[source,rust]
+----
+include::./7.rs[]
+----
+
+== Byte String Literals
+
+* not really strings
+* used to declare static bite slices (have a `&[u8]` type)
```
Comment (jonathanpallant) [inline presentations/strings/slides.adoc:130]: byte

Comment (jonathanpallant) [inline presentations/strings/slides.adoc:130]:
```suggestion
* used to declare static byte slices (have a `&[u8]` type)
```

File: presentations/strings/8.rs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+fn main() {
+    let byte_string: &[u8] = b"allows ASCII and \xF0\x9F\x98\x80 only";
+    println!("Can't be 'Display'ed {:?}", byte_string);
```
Comment (jonathanpallant) [inline presentations/strings/8.rs:3]:
```suggestion
    println!("Can Debug fmt but not Display fmt: {:?}", byte_string);
```

**Pull request merged**